WeakAurasOptions/RegionOptions/Text.lua

```diff
@@ -267,7 +291,7 @@ local function createOptions(id, data)
   };
 
   OptionsPrivate.commonOptions.AddCodeOption(options, data, L["Custom Function"], "customText", "https://github.com/WeakAuras/WeakAuras2/wiki/Custom-Code-Blocks#custom-text",
-                          37, function() return not OptionsPrivate.Private.ContainsCustomPlaceHolder(data.displayText) end, {"customText"}, false);
+                          37, hideCustomTextOption, {"customText"}, false);
```

Comment:
There's a bug in the sub text options, which is on my to do list, but since you are copying it, it's probably best if we'd fix it before.

For the custom text function, we essentially check each potential text value to check if we should show the input box.

We need to do something analog about text formatters, that's in an aura where the normal text contains no placeholders, but a condition can set the text to e.g. "%p", we should display the text formatter settings on the display text.

Essentially in the loop that adds the formatter options, also iterate all potential texts.

Reply:
Damn it's tricky to get OptionsPrivate.AddTextFormatOption grouped under same header_anchor across multiple call

Reply:
I think you need to extend AddTextFormatOption to take multiple inputs, since you want to keep the seenSymbols variable in between OptionsPrivate.Private.ParseTextStr calls.

Reply:
i got this done in commit i just pushed, but it's incomplete (explained in commit message)